app/models/subscription.rb

```diff
@@ -10,6 +10,18 @@ class Subscription < ApplicationRecord
 
   validate :qualifier_presence
 
+  def summary
+    unless name.empty?
```

Comment:
blank? works with nil (NULL), empty string, and strings that consist of only whitespace.

```suggestion
    unless name.blank?
```